docs/what-if.md

```diff
@@ -0,0 +1,5 @@
+# What If?
```

Comment:
How about adding a reference to this file from the main README.MD?

Reply:
d'oh 

![D'oh](https://m.popkey.co/e74f86/5m07l.gif)